api/src/main/java/org/pmiops/workbench/exfiltration/ObjectNameLengthServiceImpl.java

```diff
@@ -189,21 +188,16 @@ private List<BucketAuditEntry> getFileAccessInfoFromBQ() {
     return bucketAuditQueryService.queryBucketFileInformationGroupedByPetAccount();
   }
 
-  private Optional<DbEgressEvent> maybePersistEgressEvent(
+  private DbEgressEvent persistEgressEvent(
       BucketAuditEntry bucketAuditEntry, DbUser dbUser, DbWorkspace dbWorkspace) {
 
-    return Optional.of(
-        egressEventDao.save(
```

Comment:
`CrudRepository.save()` can never return null.  If there's a problem, it will throw an exception.